[Docs] Combined initial documentation work (#121)
apple/swift-distributed-tracing: Samples/Dinner/README.md

## Tracing Dinner Sample App

> In order to try this sample app, and visualize traces it produces, you should first run docker-compose in order

Reviewer: formatting nit

```suggestion
In order to try this sample app, and visualize traces it produces, you should first run `docker-compose` in order
```

> In order to try this sample app, and visualize traces it produces, you should first run docker-compose in order
> to launch a docker containers which host a Zipkin UI and collector:

Reviewer 2: ```suggestion
In order to try this sample app, and visualize the traces it produces, you should first run `docker-compose` to launch a docker container which hosts a Zipkin UI and collector:
```

```
# cd Samples/Dinner

docker-compose -f docker/docker-compose.yaml up --build
```

and then run the sample app which will produce a number of traces:

```
swift run -c release
```

> Refer to the "Trace Your Application" guide in the documentation to learn more about how to interpret this sample.

Reviewer 2: ```suggestion
Refer to the "Trace Your Application" guide to learn more about how to interpret this sample.
```

Would it be possible to link "Trace Your Application"?

> Refer to the "Trace Your Application" guide in the documentation to learn more about how to interpret this sample.

Author: Yeah okey
